packages/react-components/src/utils.ts

```diff
@@ -22,7 +22,20 @@ export const getSvgAspectRatio = (element: React.ReactElement): number => {
   return width / height;
 };
 
-export const createMailTo = (email: string): string =>
-  `mailto:${email.split('@').map(encodeURIComponent).join('@')}`;
+export const createMailTo = (
+  email: string,
+  { subject, body }: { subject?: string; body?: string } = {},
+): string => {
+  const mailtoParams = new URLSearchParams();
+  if (subject) mailtoParams.set('subject', subject);
+  if (body) mailtoParams.set('body', body);
+
+  return `mailto:${email.split('@').map(encodeURIComponent).join('@')}${
```

Comment:
Can we add a test for specifying both `body` and `subject` and preferably use the `URL` class instead of string concatenation to build this link now that it's becoming more complex (I can't tell with certainty that this works for all cases anymore)